compliance_checker/cfutil.py

```diff
@@ -2028,6 +2029,65 @@ def guess_feature_type(nc, variable):
         return "reduced-grid"
 
 
+class _CCUnit:
+    def __init__(self, units):
+        import cf_units
+
+        self.u = cf_units.Unit(units)
+        self.is_time_reference = self.u.is_time_reference()
+
+    def __eq__(self, other):
+        return self.u == other.u
+
+    def is_convertible_to(self, other):
+        if isinstance(other, str):
+            ret = self.u.is_convertible(other)
+        else:
+            ret = self.u.is_convertible(other.u)
+        return ret
+
+    def is_dimensionless(self):
+        return self.u.is_dimensionless()
+
+    def expanded(self):
+        return self.u.definition
+
+
+def _units(units: str):
+    """PLACEHOLDER."""
+    # FIXME: Make cf_units optional in a more elegant way when pyudunits2 is release.
+    try:
+        return _CCUnit(units)
+    except ImportError:
+        from pyudunits2 import UnitSystem, UnresolvableUnitException
+
+        ut_system = UnitSystem.from_udunits2_xml()
+
+    # FIXME: cf_units.Unit(None) -> Unit('unknown')
+    if units is None:
+        units = ""
+
+    # FIXME: cf_units raised only ValueError
+    try:
+        u = ut_system.unit(units)
+    except (SyntaxError, UnresolvableUnitException) as err:
+        raise ValueError from err
+    # FIXME: cf_units defined .is_time_reference for time reference units.
+    u.is_time_reference = False
+    try:
+        if hasattr(u._definition, "shift_from"):
+            u.is_time_reference = True
+    except KeyError:
+        # FIXME: hasattr should return None in that case.
+        # pyudunits2/_expr_graph.py:27, in Node.__getattr__(self, name)
+        #      25 def __getattr__(self, name):
+        #      26     # Allow the dictionary to raise KeyError if the key doesn't exist.
```

Comment:
@pelson I would expect pyudunits2 to return `False` instead of raising here. Does that make sense?